application-collabora-ui/src/main/resources/Collabora/Code/Translations.xml

```diff
@@ -53,13 +53,15 @@ collabora.attachment.modal.fileType.presentation=Presentation
 collabora.attachment.modal.fileType.spreadsheet=Spreadsheet
 collabora.attachment.modal.fileName=File name
 collabora.attachment.modal.submit=Create and edit file
+collabora.attachment.modal.submit.error=Attachment already exists
 collabora.attachment.modal.new.title=Create new file in Collabora
 collabora.attachment.modal.new=New file
 collabora.attachment.title=Collabora
 collabora.attachment.view.title=View using Collabora
 
 ## Editor
 collabora.editor.onPage=on page
+collabora.editor.error=Error while loading this file. Cause: 
```

Comment:
The ending makes me think that it's missing a parameter, but I see that you're using it in JavaScript where it's not easy to apply the parameter (on the XWiki version that you need to support).

Reply:
Ideed. Right now I added the todo comment, should be fixed after a parent upgrade